mkosi

```diff
@@ -4565,6 +4570,10 @@ def run_build_script(args: CommandLineArguments, workspace: str, raw: Optional[B
         else:
             cmdline.append("--private-network")
 
+        if args.cache_path:
+            cmdline.append("--bind=" + os.path.join(args.cache_path, "cargo") + ":/var/cache/cargo")
+            cmdline.append("--setenv=CARGO_HOME=/var/cache/cargo")
```

Comment:
Maybe just wrap the build script stuff in a `mount_cache` context? That way we don't have to add every cache in two places.